index.js

```diff
@@ -2287,6 +2319,7 @@
     filter: filter,
     reject: reject,
     map: map,
+    flip: flip,
```

Comment:
Not sure where to place the function. It seemed you have been placing derived functions underneath their underlying functions, so I inserted underneath map.

Reply:
Perfect! The order here should match the order in which the functions are defined. :)